.github/workflows/testimonial.yml

```
@@ -80,4 +80,4 @@ jobs:
         uses: YiiGuxing/close-issue@v2.1.0
         with:
           comment: |
-            Testimonial added for @${{ steps.issue-parser.outputs.issueparser_name }}. Thank you for your contribution! @${{ steps.issue-parser.outputs.issueparser_name }} if you wish to add it your LinkFree profile please follow the instructions in the docs https://linkfree.eddiehub.io/docs/how-to-guides/testimonials
+            Testimonial added for @${{ steps.issue-parser.outputs.issueparser_name }}. Thank you for your contribution! @${{ steps.issue-parser.outputs.issueparser_name }} if you wish to add it your LinkFree profile please follow the instructions in the docs https://linkfree.io/docs/how-to-guides/testimonials-json
```

We now have 2 sets of docs depending on the user's preference

```suggestion
            Testimonial added for @${{ steps.issue-parser.outputs.issueparser_name }}. Thank you for your contribution! @${{ steps.issue-parser.outputs.issueparser_name }} if you wish to add it your LinkFree profile please follow the instructions in the docs for json https://linkfree.io/docs/how-to-guides/testimonials-json or for forms https://linkfree.io/docs/how-to-guides/testimonials-forms
```